mobile/src/main/java/org/openhab/habdroid/ui/OpenHABAboutActivity.java

```diff
@@ -61,81 +91,234 @@ public void finish() {
+            ohCommunityCard.addItem(new MaterialAboutActionItem.Builder()
+                    .text(R.string.about_translation)
+                    .icon(R.drawable.ic_language_grey_24dp)
+                    .setOnClickAction(MaterialAboutItemOnClickRedirect("https://crowdin.com/profile/openhab-bot"))
+                    .build());
+            ohCommunityCard.addItem(new MaterialAboutActionItem.Builder()
+                    .text(R.string.about_foundation)
+                    .icon(R.drawable.ic_people_grey_24dp)
+                    .setOnClickAction(MaterialAboutItemOnClickRedirect("http://www.openhabfoundation.org/"))
+                    .build());
+
+            return new MaterialAboutList.Builder()
+                    .addCard(appCard.build())
+                    .addCard(ohServerCard.build())
+                    .addCard(ohCommunityCard.build())
+                    .build();
         }
 
         @Override
-        public void onResume() {
-            Log.d(TAG, "onResume()");
-            super.onResume();
+        protected int getTheme() {
+            return Util.getActivityThemeID(getActivity());
         }
-    }
 
-    public static class AboutPagerAdapter extends FragmentPagerAdapter {
-        private Context mContext;
-        private Bundle mExtras;
-
-        AboutPagerAdapter(FragmentManager fm, Context context, Bundle extras) {
-            super(fm);
-            mContext = context;
-            mExtras = extras;
+        private MaterialAboutItemOnClickAction MaterialAboutItemOnClickRedirect(final String url) {
+            return new MaterialAboutItemOnClickAction() {
+                @Override
+                public void onClick() {
+                    Uri uri = Uri.parse(url);
+                    Intent intent = new Intent(Intent.ACTION_VIEW, uri);
+                    startActivity(intent);
+                }
+            };
         }
 
-        @Override
-        public Fragment getItem(int i) {
-            switch(i) {
-                default:
-                    return new AboutFragment();
-                case 1:
-                    Fragment infoFragment = new OpenHABInfoFragment();
-                    infoFragment.setArguments(new Bundle(mExtras));
-
-                    return infoFragment;
-                case 2:
-                    return new LibsBuilder().supportFragment();
+        private String getGcmText() {
+            if (OpenHABMainActivity.GCM_SENDER_ID == null) {
+                return getString(R.string.info_openhab_gcm_not_connected);
+            } else {
+                return getString(R.string.info_openhab_gcm_connected, OpenHABMainActivity.GCM_SENDER_ID);
             }
         }
 
-        @Override
-        public int getCount() {
-            return 3;
+        private String getServerSecret(Connection conn) {
+            final String[] secret = {null};
+            conn.getSyncHttpClient().get("/static/secret", new MyHttpClient.TextResponseHandler() {
```

Comment:
I don't like the usage of the sync HTTP client here, as - depending on the server speed - it'll inevitably lead to ANR being reported.
IMHO we should show a progress indicator as long as it's loading.

Reply:
Never mind. I just saw that this method is running in an AsyncTask. Too bad, I would love to get rid of the sync HTTP client ;-)